src/utils/media.ts

```diff
@@ -0,0 +1,29 @@
+export function MediaException(message) {
+  this.message = message;
+  this.name = 'MediaException';
+}
+
+/**
+ * Get the media element when the component loads or is updated
+ */
+export function getMedia(element: string) {
+  let mediaSource = document.getElementById(element);
+
+  // Warn if no element has been set
+  if (!element) {
+    console.warn('No HTMLMediaElement was defined');
```

Comment:
No global access is allowed to `console`. Inject it.

Also every console message generated by this project should be namespaced. Make a global helper service.,